### PR TITLE
meta/add website

### PR DESCRIPTION
**Pull request by djacu**

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      };
+      packages =
+        {
+        }
+        // {
+          inherit
+            site
+            lessonsDocumentation
+            ;
+        };
```
Comment (ibizaman) [inline flake.nix:73]: Seems like you don't need the first empty attrset.

Comment (ibizaman) [reply flake.nix:73]: it will get populated later.

**Review by ibizaman — APPROVED** [review]
:tada:

File: flake.nix (hunk 1 above)
Comment (djacu) [reply flake.nix:73]: Leftovers from my other project :D

**Pull request merged**